src/test/java/com/wikia/webdriver/common/core/helpers/ContentLoader.java

```diff
@@ -0,0 +1,47 @@
+package com.wikia.webdriver.common.core.helpers;
+
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+
+/**
+ * Helper class to load content from external files
+ */
+
+public class ContentLoader {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1118)